### PR TITLE
[flutter_tools] Enable WebAssembly compilation everywhere, remove feature flag

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: packages/flutter_tools/lib/src/commands/build_web.dart [hunk 1]
```diff
@@ -148,16 +144,13 @@ class BuildWebCommand extends BuildSubCommand {
...
-      }
       if (stringArg(FlutterOptions.kWebRendererFlag) != argParser.defaultFor(FlutterOptions.kWebRendererFlag)) {
         throwToolExit('"--${FlutterOptions.kWebRendererFlag}" cannot be combined with "--${FlutterOptions.kWebWasmFlag}"');
       }
       globals.logger.printBox(
-        title: 'Experimental feature',
+        title: 'New feature',
         '''
-  WebAssembly compilation is experimental.
+  WebAssembly compilation is new. Understand the details before deploying to production.
```
Comment (yjbanov) [inline packages/flutter_tools/lib/src/commands/build_web.dart:153]: Should we link to the "details"?

**Comment by kevmoo** [comment]
Holding off until Monday to make sure the test PR sticks

File: packages/flutter_tools/lib/src/commands/build_web.dart (hunk 1 above)
Comment (kevmoo) [reply packages/flutter_tools/lib/src/commands/build_web.dart:153]: See the bits below. That's the link

**Review by eyebrowsoffire — APPROVED** [review]
LGTM!

**Pull request merged**